src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/AppModel/ContentFilePart.cs

```diff
@@ -60,7 +60,7 @@ protected override Stream GetStreamCore(FileMode mode, FileAccess access)
                 // File name will be a path relative to the applications directory.
                 // - We do not want to use SiteOfOriginContainer.SiteOfOrigin because
                 //   for deployed files the <Content> files are deployed with the application.
-                Uri codeBase = GetEntryAssemblyLocation();
+                string codeBase = GetEntryAssemblyLocation();
```

Comment:
Maybe rename the variable to something else since we don't call `CodeBase` anymore ? I would suggest using `location`:
```suggestion
                string location = GetEntryAssemblyLocation();
```